src/site/stages/build/process-cms-exports/schemas/transformed/node-press_release.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-press_release'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['press_release'] },
+    title: { type: 'string' },
+    entityMetatags: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          __typename: { type: 'string' },
+          key: { type: 'string' },
+          value: { type: 'string' },
+        },
+        required: ['__typename', 'key', 'value'],
+      },
+    },
```

Comment:
We have a common [MetaTags](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/schemas/common/entityMetaTags.js#L2) schema for this.

Reply:
That one is for the `type` version, not `__typename`.  Although I have thought of modifying it so it is `oneOf` those two versions.  Any thoughts on that?

Reply:
Oh, dang. I've been using it wrong, then. :grimacing:

Yeah, I think that `oneOf` would be good. Also maybe making those properties required on both.